scripts/tank_cmd.py

```diff
@@ -328,22 +356,13 @@ def _write_shotgun_cache(tk, entity_type, cache_file_name):
                 # deny this platform! :)
                 continue
 
-        if "title" in cmd_params["properties"]:
-            title = cmd_params["properties"]["title"]
-        else:
-            title = cmd_name
-
-        if "supports_multiple_selection" in cmd_params["properties"]:
-            supports_multiple_sel = cmd_params["properties"]["supports_multiple_selection"]
-        else:
-            supports_multiple_sel = False
+        title = cmd_params["properties"].get("title", cmd_name)
+        supports_multiple_sel = cmd_params["properties"].get(
+            "supports_multiple_selection", False)
+        deny = ",".join(cmd_params["properties"].get("deny_permissions", []))
+        icon = cmd_params["properties"].get("icon", "")
 
-        if "deny_permissions" in cmd_params["properties"]:
-            deny = ",".join(cmd_params["properties"]["deny_permissions"])
-        else:
-            deny = ""
```

Comment:
since I added the `icon` logic here, I figured I would simplify the ones around it so that they all have the same format (`.get` rather than `if/else`).